build/BUILD.bazel

```diff
@@ -92,6 +92,21 @@ wasmx_vm_cmd = select({
     "//conditions:default": "",
 })
 
+link_modules_dir = select({
+    "@kong//:nginx_dynamic_module_support": """
+    function LN {
+        if [[ "$OSTYPE" == "darwin"* ]]; then
+            # TODO: support relative path links once we start to cross compile on macOS
+            ln -sf $@
+        else
+            ln -srf $@
+        fi
+    }
```

Comment:
This is executed in the same scope as the `kong_directory_genrule` command, so it _shouldn't_ need this `LN` function to be re-defined. Can you try it without?

```suggestion
```